invenio_rdm_records/theme/templates/invenio_rdm_records/record_landing_page.html

```diff
@@ -56,12 +57,12 @@ <h1>{{ record.titles[0].title }}</h1>
 
       {# files #}
       {%- if record.files and record|can_list_files %}
-        {%- set files = record.files %}
-        {{ preview_file_box(files|select_preview_file, pid) }}
-        {{ file_list_box(files.dumps(), pid) }}
+      {%- set files = record.files %}
+      {{ preview_file_box(files|select_preview_file, pid) }}
+      {{ file_list_box(files.dumps(), pid) }}
```

Comment:
does the indentation cause problem on the HTML? Otherwise, I would keep it as it's more readable IMHO.